apps/web/src/app/[locale]/shop/_auction/SellSection/SellInputRow.tsx

```
@@ -102,6 +132,71 @@ function SellInputRow({ item, initPersona }: Props) {
+  const t = useTranslations('Shop');
+
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <Dialog.Content>
+        <Dialog.Title>{t('sell-price-modal-title')}</Dialog.Title>
+        <input
+          className={mobilePriceinputStyle}
+          placeholder="Type price..."
+          type="number"
+          value={Boolean(sellPrice) ? sellPrice : ''}
+          onChange={(e) => setSellPrice(Number(e.target.value))}
```

Biome (1.9.4)

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)